build/flaws/index.ts

```diff
@@ -222,7 +222,8 @@ export async function fixFixableFlaws(doc, options, document) {
           );
           console.log(`Downloaded ${flaw.src} to ${destination}`);
           newSrc = path.basename(destination);
-        } catch (error) {
+        } catch (e) {
+          const error = e as RequestError;
```

Comment:
Rather than typecast to `RequestError`, can we instead check if the error is a `RequestError` and use a generic fallback if it's not?

Reply:
Good catch, I hadn't double-checked and assumed that this `RequestError` is just a TypeScript interface.

Fixed in 6bea38816.